src/dev-app/mdc-button/mdc-button-demo.html

```diff
@@ -76,6 +86,11 @@ <h4 class="demo-section-header">Stroked Buttons [mat-stroked-button]</h4>
     <button mat-stroked-button color="accent">accent</button>
     <button mat-stroked-button color="warn">warn</button>
     <button mat-stroked-button disabled>disabled</button>
+    <button mat-stroked-button>
+      <mat-icon>home</mat-icon>
+      with icons
+      <mat-icon iconPositionEnd>favorite</mat-icon>
```

Comment:
Should the selector be prefixed with `mat`?

Reply:
This is a pre-existing API, I'm just using it in this demo